CHANGELOG.md

```diff
@@ -3418,3 +3418,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [0.2.x] - 2019-07-09
 
 ## [0.1.x] - 2019-06-DD
+
+### Fixed
+
+- Rearranged documentation for 3rd party loggers.
+
+### Added
+- `%%wandb` magic command to the WandbLogger usage
```

Comment:
1. docs changes typically don't require changelog entries
2. this is the oldest section in the changelog - newer updates should be added at the top